README.md

```diff
@@ -202,7 +261,7 @@ about the software that makes workflowr possible.
 
 ## License
 
-workflowr is available under the [MIT][] license.
+workflowr is available under the [MIT license][].
```

Comment:
Moving license into the hyperlink breaks the markdown reference. It should be specified as `[MIT license][MIT].`